tests/main/postrm-purge/task.yaml

```diff
@@ -95,7 +95,11 @@ execute: |
 
     # XXX: use retry instead
     sleep 5
-    systemctl --no-legend --full > output.txt
+    # snap services may have failed and systemd will keep their 'state' in
+    # memory even after it's reloaded, since the purge does not run reset-failed
+    # to allow the user to inspect them as need we need to account for those
+    # entries in the output
+    systemctl --plain --no-legend --full |grep -v ' not-found failed ' > output.txt
```

Comment:
Is this sensitive to locale?

Reply:
we're setting locale here: https://github.com/snapcore/snapd/blob/b04ff60f9b12b792259c066a243c855e8f879173/spread.yaml#L38-L39

Reply:
Thanks, I think it looks good.